recipe/meta.yaml

```diff
@@ -1,15 +1,22 @@
 {% set version = "18.1.8" %}
 
+{% if not variant %}
+{% set variant = "default" %}
+{% endif %}
+
 package:
   name: flang-split
   version: {{ version }}
 
 source:
   url: https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-{{ version.replace(".rc", "-rc") }}.tar.gz
   sha256: 09c08693a9afd6236f27a2ebae62cda656eba19021ef3f94d59e931d662d4856
+  patches:
+    - patches/wasm/01-Minimal-WASM-support-for-flang.patch  # [variant and variant.startswith("emscripten_")]
+    - patches/wasm/02-Specialize-Flang-to-target-WASM.patch  # [variant and variant.startswith("emscripten_")]
```

Comment:
Can you please document where these patches are coming from (e.g. url to a fork/branch or whatever)?